Packs/MailListener/Integrations/MailListenerV2/MailListenerV2.py

```diff
@@ -0,0 +1,220 @@
+from datetime import timezone
+from typing import Any, Dict
+
+from dateparser import parse
+from mailparser import parse_from_bytes, MailParser
+from imap_tools import OR
+from imapclient import IMAPClient
+
+import demistomock as demisto
+from CommonServerPython import *  # noqa: E402 lgtm [py/polluting-import]
```

Comment:
i dont think we need `# noqa: E402 lgtm [py/polluting-import]`
please check if lint passes if you remove it